Add minimal sql support (#144)
tarantool/go-tarantool: README.md

}
```

> For use SQL to query to tarantool, call `Execute`. 

Reviewer: I use Google Doc to verify syntax, it's really useful.

"To use SQL to query a tarantool instance, call `Execute`."

Author: Thank you for advice, I will use it onward! Fixed.

> To use SQL to query a tarantool instance, call `Execute`. 

Reviewer 2: Trailing space.

Author: Removed.

> Pay an attention, that with the different types of query (DDL, DQL, DML etc.),

Reviewer: "Pay attention that with different types of queries (DDL, DQL, DML etc.) some fields of the response structure (`MetaData` and `SQLInfo`) may be empty."

Author: Fixed.

> Pay attention that with different types of queries (DDL, DQL, DML etc.) 

Reviewer 2: Trailing space.

Author: Removed.

> some fields of the response structure (`MetaData` and `SQLInfo`) may be empty.

Reviewer: Empty means nil?

Author: Yes, fixed.

See the [protocol](https://www.tarantool.io/en/doc/latest/dev_guide/internals/box_protocol/#responses-for-sql) explanation for details.

> ```go

Reviewer 2: Please don't put example to the README. Put it to tests instead. Other examples I'll move in scope of #123.

See [Testable Examples in Go](https://go.dev/blog/examples) and https://pkg.go.dev/testing#hdr-Examples and existed examples in https://github.com/tarantool/go-tarantool/blob/master/example_test.go

Author: Fixed.

package main

import (
	"github.com/tarantool/go-tarantool"
	"log"
	"time"
)

func main() {
  server := "127.0.0.1:3013"
  opts := tarantool.Opts{
    Timeout:       500 * time.Millisecond,
    Reconnect:     1 * time.Second,
    MaxReconnects: 3,
    User:          "test",
    Pass:          "test",
  }
  client, err := tarantool.Connect(server, opts)
  if err != nil {
    log.Fatalf("Failed to connect: %s", err.Error())
  }
  

>   resp, err := client.Execute("CREATE TABLE SQL_TEST (id INTEGER PRIMARY KEY, name STRING)")

Reviewer 2: Trailing whitespace.

Author: Removed everything about SQL from Readme.

  log.Println("Execute")
  log.Println("Error", err)
  log.Println("Code", resp.Code)
  log.Println("Data", resp.Data)
  log.Println("MetaData", resp.MetaData)
  log.Println("SQL Info", resp.SQLInfo)

>   // there are 3 options to give named parameters in SQL query

Reviewer: ```suggestion
  // there are 3 options to pass named parameters to an SQL query
```

Author: Fixed.

    "id":   1,
    "name": "test",
  }

  sqlBind2 := struct {
    Id   int
    Name string
  } {1, "test"}

  type kv struct {
    Key   string
    Value interface{}
  }

  sqlBind3 := []kv {
    kv{"id", 1},
    kv{"name", "test"},
  }

  // the next usage
  resp, err = client.Execute("SELECT id FROM SQL_TEST WHERE id=:id AND name=:name", sqlBind1)
  log.Println("Execute")
  log.Println("Error", err)
  log.Println("Code", resp.Code)
  log.Println("Data", resp.Data)
  log.Println("MetaData", resp.MetaData)
  log.Println("SQL Info", resp.SQLInfo)

  // the same as
  resp, err = client.Execute("SELECT id FROM SQL_TEST WHERE id=:id AND name=:name", sqlBind2)
  log.Println("Execute")
  log.Println("Error", err)
  log.Println("Code", resp.Code)
  log.Println("Data", resp.Data)
  log.Println("MetaData", resp.MetaData)
  log.Println("SQL Info", resp.SQLInfo)

  // the same as
  resp, err = client.Execute("SELECT id FROM SQL_TEST WHERE id=:id AND name=:name", sqlBind3)
  log.Println("Execute")
  log.Println("Error", err)
  log.Println("Code", resp.Code)
  log.Println("Data", resp.Data)
  log.Println("MetaData", resp.MetaData)
  log.Println("SQL Info", resp.SQLInfo)

>   // there are 2 options to give positioned arguments to SQL query

Reviewer: ```suggestion
  // there are 2 options to pass positional arguments to an SQL query
```

Author: Fixed.
